config/auto_battle_operation/般岳-普通攻击-摧岳.sample.yml

```
@@ -0,0 +1,9 @@
+# 般岳-普通攻击-摧岳（连续EX后终结）
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)